commandovm.win10.preconfig.fireeye/tools/README.md

```diff
@@ -1,3 +1,15 @@
+## The project is archived
+
+This project has been archived because I no longer use Windows. I grew tired with the system being unable to keep itself configured in the desired state, group policies randomly stopping working for hundreds of managed workstations at once, advertisements, unwanted tips and content popping up on various places throughout the user interface, and incompetent support even for enterprise products. Ultimately I have migrated virtually all my servers, workstations and laptops to linux. Even though there are areas where the open source products are still lacking, the transparency, configurability and reliability greatly outweighs the drawbacks for me.
+
+I also lost faith in humanity throughout the years I was actively maintaining the project. The influx of pings, bumps, repeating questions already answered in FAQ, and various other unreasonable demands without any givebacks was just astounding. Please do not contact me regarding this project as I have no desire to invest any more time into it. As per MIT license, you're still free to use, modify, and distribute the code further, as long as you keep the license and copyright notice.
```

Comment:
yes... big sad

Reply:
in case anyone out there is reading this, Commando is not abandoned lol.. this is from the https://github.com/Disassembler0/Win10-Initial-Setup-Script repo which we use to pre-configure Commando